tests/test_pic_merge.py

```diff
@@ -0,0 +1,114 @@
+import unittest
+import shutil
+import os
+import picturae_csv_create as pcc
+from datetime import date, timedelta
+from tests.testing_tools import TestingTools
+
+def test_date():
+    """test_date: creates an arbitrary date, 20 years in the past from today's date,
+       to create test files for, so as not to overwrite current work
+       ! if this code outlives 20 years of use I would be impressed"""
+    unit_date = date.today() - timedelta(days=365 * 20)
+    return str(unit_date)
+
```

Comment:
If this is against a test database, we don't need to uniquely idenfify data, do we? all data would be test data.